lib/presentation/pages/trip_panel_page/widgets/rounded_icon.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:flutter/material.dart';
+import 'package:trip_tally/presentation/theme/app_colors.dart';
+import 'package:trip_tally/presentation/theme/app_dimensions.dart';
+import 'package:trip_tally/presentation/widgets/m3_widgets/svg_asset.dart';
+
+class RoundedIcon extends StatelessWidget {
+  const RoundedIcon({required this.iconPath, super.key});
+
+  final String iconPath;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
```

Comment:
You can simplify this by using `CircleAvatar` and removing hardcoded height, width:

`CircleAvatar(
            backgroundColor: context.thc.primaryContainer,
            child: SvgAsset(
              TransportType.flight.icon,
              color: context.thc.shadow,
            ),
          ),`